products/gerrit.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Gerrit Code Review
+category: server-app
+iconSlug: "gerrit"
+permalink: /gerrit
+versionCommand: "java -jar gerrit.war version"
+releasePolicyLink: https://www.gerritcodereview.com/support.html
+changelogTemplate: https://www.gerritcodereview.com/__RELEASE_CYCLE__.html#{{"__LATEST__"| replace:'.',''}}
+activeSupportColumn: true
+releaseColumn: true
+releaseDateColumn: true
+
+identifiers:
+-   purl: pkg:docker/gerritcodereview/gerrit
+
+auto:
+-   git: https://github.com/GerritCodeReview/gerrit.git
+    regex: ^v(?<major>[0-9]+)\.(?<minor>[0-9]+)\.(?<patch>[0-9]+)(\.(?<tiny>[0-9]+))?$
```

Comment:
This is missing .0 releases, which don't have a patch component: https://rubular.com/r/0BqxymSxb3nVwE

marking patch as optional fixes it: https://rubular.com/r/b1ZtGZ98I7vc5A

```suggestion
    regex: ^v(?<major>[0-9]+)\.(?<minor>[0-9]+)(\.(?<patch>[0-9]+)(\.(?<tiny>[0-9]+))?)?$
```

Reply:
done